vapid.js

```diff
@@ -32,10 +48,10 @@ var vapid = {
            .then(keys => {
               this._private_key = keys.privateKey;
               this._public_key = keys.publicKey;
-              console.info("VAPID Keys defined.");
+              console.info(this.errs.enus.OK_VAPID_KEYS);
```

Comment:
since some of these messages are put into `console.info` statements, should we call it `this.console_messages` instead of `console.errs.enus`?